kafka-commons/build.gradle

```diff
@@ -3,7 +3,7 @@ ext {
 }
 
 dependencies {
-    compile "mesosphere:dcos-commons:0.7.12"
+    compile "mesosphere:dcos-commons:0.8.1"
```

Comment:
It'd make sense to just go with `0.8.1.1` which was recently released to include this fix: https://github.com/mesosphere/dcos-commons/commit/8c493ceceb00ec32d2a7eadd7823e696deafccf8